### PR TITLE
Importing addons

### PR DESCRIPTION
**Pull request by tsl143**
Feature to let user import addons from FIrefox to Cliqz.

### CODE REVIEW

File: mozilla-release/browser/components/migration/FirefoxProfileMigrator.js [hunk 1]
```diff
@@ -511,11 +551,14 @@ FirefoxProfileMigrator.prototype._getResourcesInternal = function(sourceProfileD
   }
 
   let types = MigrationUtils.resourceTypes;
+
```
Comment (alver-cliqz) [inline mozilla-release/browser/components/migration/FirefoxProfileMigrator.js]: please, remove empty spaces in this file

File: mozilla-release/browser/branding/cliqz/pref/firefox-branding.js [hunk 2]
```diff
@@ -45,3 +45,4 @@ pref("browser.aboutHomeSnippets.updateUrl", "https://www.cliqz.com/");
 // Cliqz distribution settings
 pref("app.distributor", "cliqz");
 pref("mozilla.partner.id", "cliqz");
+pref("extensions.cliqz.integrated", "cliqz@cliqz.com,firefox@ghostery.com,https-everywhere@eff.org");
```
Comment (luciancor) [inline mozilla-release/browser/branding/cliqz/pref/firefox-branding.js]: this list can grow, can't we read them somehow? And ghostery is not always in, the user needs to install it manually from the prefs page

Comment (luciancor) [reply mozilla-release/browser/branding/cliqz/pref/firefox-branding.js]: plus can you please use a different namespace, other than extension.cliqz? maybe "browser.migration.soemthing"

Comment (tsl143) [reply mozilla-release/browser/branding/cliqz/pref/firefox-branding.js]: @luciancor, This list is not part of migration but allowing install from AMO, so IMO the namespace suits.
We are planning to get rid of installs from prefs, so probably we will not have Ghostery in the list. As I understand this list is of extensions pre-installed in browser and supposedly should not grow. We have separate feature `blocklist` which will take care of addons which will not be allowed to install.

**Comment by tsl143** [comment]
@alver-cliqz , formatted all(hopefully) the code in PR.

File: mozilla-release/browser/components/migration/content/migration.js [hunk 3]
```diff
@@ -185,7 +186,8 @@ var MigrationWizard = { /* exported MigrationWizard */
     // check for more than one source profile
     var sourceProfiles = this.spinResolve(this._migrator.getSourceProfiles());
     if (this._skipImportSourcePage) {
-      this._wiz.currentPage.next = "migrating";
+      // Trishul: check if no addons, then move to migrate page directly.
```
[the remark this answers is not in the document]
Comment (aleksei-cliqz) [reply mozilla-release/browser/components/migration/content/migration.js]: @tsl143 If you decide using "CLIQZ" then please use "CLIQZ-SPECIAL" for the same style everywhere.
"Grepping" by CLIQZ-SPECIAL will allow us to find the most hard and interesting parts of code which probably we need to start with to fix / improve anything.

[the remark this answers is not in the document]
Comment (tsl143) [reply mozilla-release/browser/components/migration/content/migration.js]: @aleksei-cliqz @alver-cliqz , I changed this to 'CLIQZ-TODO' in current version

**Pull request merged**